### PR TITLE
Fix #112 Fix #114 - Ohm Signs

### LINKED ISSUE #112: Inverted Ohm Sign have wrong design except Libertinus Serif Regular
Inverted Ohm Sign (U+2127) in Libertinus Serif become higher while switching to Bold or Italic faces, and it looks not bolded in Bold Italic face. Additionaly, it will overlap with precedding character in Libertinus Sans Italic, Bold and Bold Italic faces.

### LINKED ISSUE #114: Ohm Sign not bolded in Libertinus Serif Bold Italic
In Libertinus Serif, when I switch to Bold and Italic face, Ohm sign (U+2126) looks the same as Libertinus Serif Italic.

See: #112

### PR DESCRIPTION
**Pull request by mgavioli**
Fixes https://github.com/khaledhosny/libertinus/issues/114
Fixes https://github.com/khaledhosny/libertinus/issues/112

Summary
As several fonts already had the Ohm sign as a reference to U+03A9 (Greek Omega), in all fonts both Ohm and Inv. Ohm signs have been replaced with references (horiz.+vert. flipped for Inv. Ohm) to U+03A9.

In addition, all side bearing have been checked and adjusted (increased) when necessary and the vertically misplaced signs have been brought to the base line.

Detail

**Sans**

**Regular**:
- Ohm: increased right side bearing
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9` (Greek Omega), revised side bearings

**Italic**:
- Ohm: increased right side bearing
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`. revised side bearings

**Bold**:
- Ohm: increased right side bearing
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`. revised side bearings

**Serif**

**Regular**:
- Ohm: no change
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9` (Greek Omega)

**Italic**:
- Ohm: replaced with ref. to `U+03A9` and revised side bearings
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`, lowered to base line and revised side bearings

**Semi-bold**:
- Ohm: revised right side bearing
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`, lowered to base line and revised side bearings

**Semi-bold italic**:
- Ohm: replaced with ref. to `U+03A9` and revised side bearings
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`, lowered to base line and revised side bearings

**Bold**:
- Ohm: revised side bearings
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`, lowered to base line and revised side bearings

**Bold italic**:
- Ohm: replaced with ref. to `U+03A9` and revised side bearings
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`, lowered to base line and revised side bearings

**Display**:
- Ohm: replaced with ref. to `U+03A9`
- Inv. Ohm: replaced with h+v flipped ref. to `U+03A9`

### CODE REVIEW

**Comment by khaledhosny** [comment]
You should set “Use My Metrics” flag for all the references to keep the advance width in sync.

File: sources/libertinussans-bold.sfd [hunk 1]
```diff
@@ -22906,39 +22906,11 @@ EndChar
...
- 224 488 l 1
- 142 447 111 382 111 274 c 0
- 111 133 167 42 297 42 c 0
- 436 42 483 145 483 273 c 0
- 483 385 462 426 370 488 c 1
- 362 636 l 2
- 362 645 373 651 384 651 c 2
- 549 651 l 2
-EndSplineSet
+Refer: 704 937 S -1 0 0 -1 765.999 651 2
```
Comment (khaledhosny) [inline sources/libertinussans-bold.sfd]: Hmm, the references are selected but `make check` didn’t catch that!

Comment (khaledhosny) [reply sources/libertinussans-bold.sfd]: Bug in the normalizer, please rebase on master and normalize again.

Comment (mgavioli) [reply sources/libertinussans-bold.sfd]: Ok, master updated, I'll rebase and re-normalise.

**Comment by mgavioli** [comment]
> You should set “Use My Metrics” flag for all the references to keep the advance width in sync.

In synch with what? With the original Greek glyph? This is not always what I want, for sure in the inverted Ohm sign, which needs its own side bearing (particularly in the various Italics), but in general, I preferred to re-check the spacing of each glyph.

Finally, according to FF own tooltip, the "Use my metrics" check is only relevant in TrueType fonts.

**Comment by mgavioli** [comment]
Rebased and re-normalised.

**Comment by khaledhosny** [comment]
> Finally, according to FF own tooltip, the "Use my metrics" check is only relevant in TrueType fonts.

Again, for CFF fonts this is an editor thing, it helps keeping advance width in sync, you can still have different side bearings, only the advance width is locked.

**Pull request merged**